website/docs/docs/dbt-versions/release-notes/03-Oct-2023/api-v2v3-limit.md

```diff
@@ -0,0 +1,15 @@
+---
+title: "API results limited to `100`"
+id: apiv3-limit"
+description: "Oct 2023: In order to enhance the efficiency and stability of our services, we will limit all API results to `100` records. This limit is applicable to multi-tenant instances only."
+sidebar_label: "Update: API results limited to `100`"
+sidebar_position: 04
+tags: [Oct-2023, API]
+---
+
+
+Beginning December 1, 2023, the [Administrative API](/docs/dbt-cloud-apis/admin-cloud-api) v2 and v3 will limit all "list" or `GET` API methods to 100 results per API request. This limit enhances the efficiency and stability of our services. You can handle more than 100 results by using the `limit` and `offset` query paramters to paginate those results. Otherwise, exceeding the limit of `100` will cause an error. 
```

Comment:
A little unclear on the last part "Otherwise, exceeding the limit of `100` will cause an error. " Should they expect to get an error if they make an API call that returns more than 100 results? The first part of this makes it seem like they'll just get a truncated list of 100 results.

Reply:
Yes, current behavior is we'll return an error (`400`) with a message saying the limit is too large. This is to prevent issues with the usage of larger limits without realizing they need to change the offset they're passing in for paginated results. For example. if a user was passing limit=1000 and paginating forward 1000 items at a time and we implemented this change without them realizing, rather than breaking their integration they'd end up with 900 missing results in their list